cirq/contrib/qcircuit/qcircuit_pdf.py

```diff
@@ -50,6 +50,10 @@ def circuit_to_pdf_using_qcircuit_via_tex(circuit: circuits.Circuit,
     doc = Document(documentclass=documentclass, document_options='dvips')
     doc.packages.append(Package('amsmath'))
     doc.packages.append(Package('qcircuit'))
+    try:
+        tex.encode('ascii')
+    except UnicodeEncodeError:
+        doc.preamble.append(Package('inputenc', options=['utf8']))
```

Comment:
Let's just add this unconditionally. (Is there a downside?) It will make the behavior of the code more consistent, and make it easier to cover the code in tests.

Reply:
Done! I agree this is cleaner, the only reason why I added that was because you folks mentioned it in the ticket.